server/chat-commands/moderation.js

```diff
@@ -1415,7 +1417,7 @@ exports.commands = {
 		} else {
 			room.send(`|c|~|${name}'s messages were cleared from ${room.title} by ${user.name}.`);
```

Comment:
hyper nit - maybe this should say if they're only clearing the last n lines?

Reply:
Sure, that's easy enough.